.github/workflows/container.yml

```
@@ -27,16 +27,16 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ github.token }}
 
-      - name: Set lower case owner name
-        env:
-          OWNER: '${{ github.repository_owner }}'
+      - name: Find the registry owner to use
         run: |
-          echo "OWNER_LOWERCASE=${OWNER,,}" >>${GITHUB_ENV}
+          OWNER="${{ github.repository_owner }}"
+          [[ "${{github.event_name}}" == "pull_request" ]] && OWNER="${{ github.actor }}"
```

I think it'd prefer the boring, obvious if-then-else here:

```suggestion
          if [[ "${{github.event_name}}" == "pull_request" ]]; then
              OWNER="${{ github.repository_owner }}"
          else
              OWNER="${{ github.actor }}"
          fi
```

---

It's the other way around, but that's maybe a good argument for doing it the more normal way.